create/how_to/digitalocean_droplet.md

```diff
@@ -11,69 +11,69 @@
 A "droplet" is a set of resources, as a Virtual Machine, or a Server, in which you can run your own applications.
 In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on "Create" -> "Droplets".
 
-![Create droplet](/digitalocean/01.create.png)
+![Selecting Droplets Create cloud servers](/digitalocean/01.create.png)
 
 ### 2. Select MeiliSearch snapshot
 
 By default, DigitalOcean will display the "distributions" tab. Select the "Marketplace" tab and search for "meili". Select it.
 
-![Marketplace](/digitalocean/02.marketplace.png)
+![Searching for MeiliSearch in Marketplace](/digitalocean/02.marketplace.png)
 
 ### 3. Select your plan
 
 Select your plan. Plans start at $5 (click on "See all plans" for more options). Memory-optimized options will give you better results for a production environment on big datasets.
 
-![Select plan](/digitalocean/03.select-plan.png)
+![Selecting the $5/month plan](/digitalocean/03.select-plan.png)
```

Comment:
Should we mention the price of the plan? Perhaps we can future-proof the alt text a bit by not mentioning it.

Reply:
Good idea. But how do I identify the plan selected in the image? 🤔

Reply:
Perhaps by calling it "the cheapest plan"? Or perhaps we can get by without mentioning a specific plan? Like, as far as setting up MeiliSearch goes, it doesn't really matter which plan the user chooses.